components/molecule/textareaField/src/index.js

```diff
@@ -135,7 +147,10 @@ MoleculeTextareaField.propTypes = {
   disabled: PropTypes.bool,
 
   /** Boolean to decide if helptext should be auto hide */
-  autoHideHelpText: PropTypes.bool
+  autoHideHelpText: PropTypes.bool,
+
+  /** Prop to handle if the user can exceed the maxChars length  */
+  canExceedLenth: PropTypes.bool
```

Comment:
misspelling error
better use isMaxCharBlocked, and default true for retrocompatibility

Reply:
isMaxCharOvercomed

Reply:
semantical afirmative boolean prop is prefixed

Reply:
But if. we set it to true we may break people using it since we're going to force them to add the new prop, since it will always exceed length.

Reply:
dnt know what is happening right now.

Its able to overcome? --> true
Its not? --> false

Reply:
Ok so i will change name and set it to false by default!